specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -4060,6 +4121,38 @@
         }
       }
     },
+    "BillingProfileCreationParameters": {
+      "description": "The parameters for creating a new billing profile.",
+      "properties": {
+        "displayName": {
+          "description": "The billing profile name.",
+          "type": "string"
+        },
+        "poNumber": {
+          "description": "Purchase order number.",
+          "type": "string"
+        },
+        "address": {
+          "description": "Billing address.",
+          "$ref": "#/definitions/Address"
+        },
+        "invoiceEmailOptIn": {
+          "description": "If the billing profile is opted in to receive invoices via email.",
+          "type": "boolean"
+        },
+        "enableAzureSKUs": {
+          "description": "Azure skus to enable for this billing profile..",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EnabledAzureSKUs"
+          }
+        },
+        "createAzureSubscriptions": {
```

Comment:
what does this do? Does it create one subscription under that profile? Lets make sure this leverages the new PUT way of creating azure subs and if any input is needed from the caller for the subscription, lets get that as part of the payload.
How will authorization work in this case? Does billingProfile write permission automatically give them subscription provisioning permission? Or can it be that the customer can set it up in a way where someone can create billing profile but not subscriptions

Reply:
yes this will create subscriptions for the user under the sku rating asset, if they choose to. The user  can choose not to create the subscription at the time of creating the billing profile. Only the rating asset will be created. the user can then later on create a subscription under that rating asset.

Reply:
removing CreateAzureSubscriptions request field. This api will create a billing profile without subs. Users can then call the subscriptions api to create a subscription under this billing profile.